### ISSUE TITLE
vaughnlive.tv - 503 server error again

### ISSUE THREAD

**Issue by fadster** [chrippa/livestreamer#1297]
$ ./livestreamer http://vaughnlive.tv/lmshows_ngbbc
[cli][info] Found matching plugin vaughnlive for URL http://vaughnlive.tv/lmshows_ngbbc
error: Unable to open URL: http://vaughnlive.tv/lmshows_ngbbc (503 Server Error: Service Temporarily Unavailable)

I've been trying for 15 minutes with the same error result. :(

**Comment by fadster** [comment]
I just placed the command in a loop and it eventually worked. If anybody knows what's causing this to happen from time to time, I would love to know. Thanks.

**Comment by fadster** [comment]
I'm reopening the issue because it's happening much more frequently and only clears up after quite a while. So far, when I get this error, it takes up to half an hour to eventually work.

**Comment by L2501** [comment]
vaughnlive limits the number of free viewers on streams

**Comment by fadster** [comment]
Yes but this limit didn't seem to affect livestreamer until two days ago. The player on vaughnlive's site would give capacity exceeded messages while livestreamer would play with no issues, but something appears to have changed in the last two days. Also, while I'm getting these 503 errors with livestreamer, I am able to play the stream through the web player, which makes me think the 503 is not directly related to that limit.
